tajo-catalog/tajo-catalog-drivers/pom.xml

```diff
@@ -41,6 +41,7 @@
         <configuration>
           <excludes>
             <exclude>tajo-hive/**</exclude>
+            <exclude>tajo-hcatalog/**</exclude>
```

Comment:
This looks to be already removed in https://issues.apache.org/jira/browse/TAJO-1442.